docs/2_step_by_step/2_6_start_finetune.md

```diff
@@ -1 +1,104 @@
-# Start fine-tuning
\ No newline at end of file
+# Start fine-tuning
+
+Now you should have your training data and evaluation data (optional) prepared as `DocumentArray`,
+and have decided your backbone model.
+
+To start fine-tuning, you can call:
+
+```python
+import finetuner
+from docarray import DocumentArray
+
+train_data = DocumentArray(...)
+
+run = finetuner.fit(
+    model='efficientnet_b0',
+    train_data=train_data
+)
+print(f'Run name: {run.name}')
+print(f'Run status: {run.status}')
+```
+
+You'll see this in the terminal:
+
+```bash
+Run name: vigilant-tereshkova
+Run status: Created
+```
+
+During fine-tuning,
+the run status changes from:
+1. Created: the Run has been created and submitted to the job queue.
+2. Started: the job is in progress
+3. Finished: the job finished successfully, model has been sent to cloud storage.
+4. Failed: the job failed, please check the logs for more details.
+
+Beyond the simplist use case,
+Finetuner gives you the flexibility to set hyper-parameters by yourself:
+
+```python
+import finetuner
+from docarray import DocumentArray
+
+train_data = DocumentArray(...)
+eval_data = DocumentArray(...)
+
+run = finetuner.fit(
+    model='efficientnet_b0',
+    train_data=train_data,
+    eval_data=eval_data, 
+    run_name='finetune-flickr-dataset-efficientnet-1',
+    description='this is a trial run on flickr8k dataset with efficientnet b0.',
+    experiment_name='finetune-flickr-dataset',
```

Comment:
Since you're using `experiment_name` here, I think we should show that experiment must be created first. 
```python
finetuner.create_experiment('finetune-flickr-dataset')
```